src/DiffEngine/DiffTools.cs

```diff
@@ -255,14 +263,30 @@ public static void UseOrder(in bool throwForNoTool, params DiffTool[] order)
         return ExtensionLookup.TryGetValue(extension, out tool);
     }
 
+    [Obsolete("Replaced with TryFindByName. Made obsolete in version 8.5")]
     public static bool TryFind(
         DiffTool tool,
         [NotNullWhen(true)] out ResolvedTool? resolvedTool)
+    {
+        return TryFindByName(tool, out resolvedTool);
+    }
+
+    public static bool TryFindByName(
```

Comment:
To discuss: maybe "name" is wrong here. 

Maybe leave it to TryFind or rename it to something else.

Reply:
i am happy with "name"